src/vivarium/framework/lookup/manager.py

```diff
@@ -156,11 +151,38 @@ def _build_table(
     def __repr__(self) -> str:
         return "LookupTableManager()"
 
+    @staticmethod
+    def _get_columns(
+        value_columns: list[str] | tuple[str, ...] | str, data: pd.DataFrame
+    ) -> tuple[list[str], list[str]]:
+        if isinstance(value_columns, str):
+            value_columns = [value_columns]
+
+        all_columns = list(data.columns)
+
+        potential_parameter_columns = [
+            str(col).removesuffix("_start")
+            for col in all_columns
+            if str(col).endswith("_start")
+        ]
+        parameter_columns = []
+        bin_edge_columns = []
+        for column in potential_parameter_columns:
+            if f"{column}_end" in all_columns:
```

Comment:
Would there ever *not* be the corresponding column_end column? What would that even mean?